ui/components/multichain/account-list-item/account-list-item.test.js

```diff
@@ -22,7 +23,10 @@ const mockAccount = {
 };
 
 const mockNonEvmAccount = {
-  ...mockAccount,
+  ...mockState.metamask.internalAccounts.accounts[
+    'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3'
+  ],
+  balance: '1', // updating this  to 1 because the balance for native non evm networks comes from the multichainBalances controller in decimal format and not hex.
```

Comment:
Selector where we set balance inside the account object is here https://github.com/MetaMask/metamask-extension/blob/c55d167c46714587a31e21e920fbad4c6a95079e/ui/selectors/selectors.js#L393
I think this test is very old; i expect all non EVM accounts to have balances coming from multichain controller; hence in decimal format.